api/src/configuration.ts

```diff
@@ -0,0 +1,18 @@
+import { nbVal, strVal } from "@paroi/data-formatters-lib";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+export const appConfig = {
+  thisServer: {
+    port: nbVal(process.env.PORT),
+  },
+  database: {
+    url: strVal(process.env.MONGODB_URI),
+  },
+  redis: {
+    host: strVal(process.env.REDIS_HOST),
```

Comment:
```suggestion
    host: String(process.env.REDIS_HOST),
```
n'aurait pas suffit ?

Reply:
Ce sont des helpers qui font un checking de l'existence et du typage de la variable d'environnement. En utilisant Numéro ou String ça ne vérifie pas l'existence de la variable

Reply:
@LewisYann